graphql-kotlin-federation/pom.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.expedia</groupId>
+        <artifactId>graphql-kotlin</artifactId>
+        <version>1.0.0-RC2-SNAPSHOT</version>
+        <relativePath>..</relativePath>
+    </parent>
+
+    <artifactId>graphql-kotlin-federation</artifactId>
+    <name>graphql-kotlin-federation</name>
+    <packaging>jar</packaging>
+
+    <build>
+        <sourceDirectory>${project.basedir}/src/main/kotlin</sourceDirectory>
+        <testSourceDirectory>${project.basedir}/src/test/kotlin</testSourceDirectory>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
```

Comment:
Does this do something I cannot find?

Reply:
It applies detekt and ktlint plugins from the parent (https://github.com/ExpediaDotCom/graphql-kotlin/blob/master/pom.xml#L138). Cannot apply them at the root project as it doesn't have any source code and there are no official maven plugins that would work out of the box.